nss-day-cohort-48/rare-client-polite-pirates
# create category

#### Changes Made
1. Modified file `CategoryList.js` to include `create button` and correct path on click.
2. Modified file `CategoryForm.js` to include `create form fields`.
   
#### Steps to Review
1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-create-category
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
- [x] run server and client in terminal
- [x] go to `Category Manager` to create a category
- [x] newly created category will appear on main view

4. View code file.
   > Confirm file modifications are present as indicated above.
